.github/workflows/mac-artifact.yml

```
@@ -16,7 +16,10 @@ on:
 
 jobs:
   build:
-    runs-on: macos-11
+    strategy:
+      matrix:
+        os: [macos-11, macos-14]  
```

```suggestion
        os: [macos-12, macos-14]  
```
Macos-11 runners are no longer supported.